applications/GeoMechanicsApplication/custom_utilities/check_utilities.h

```diff
@@ -27,5 +30,10 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) CheckUtilities
     static void CheckDomainSize(double                            DomainSize,
                                 std::size_t                       Id,
                                 const std::optional<std::string>& PrintName = std::nullopt);
+								
+	static void CheckNodalVariables(const Geometry<Node>&            rGeometry,
+                                    const std::vector<VariableData>& rVariables);
+    static void CheckNodalDof(const Geometry<Node>&            rGeometry,
```

Comment:
Also here, we may want to use a slightly more specific name (since we only check whether the given degrees of freedom are present):
```suggestion
    static void CheckHasDofs(const Geometry<Node>&            rGeometry,
```

Reply:
changed name